src/components/Graph.js

```diff
@@ -399,7 +399,7 @@ export class Graph extends React.Component {
       nodeCircle: nodeCircle,
       nodeIcon: nodeIcon,
       nodeGroup: nodeGroup,
-      showLabels: !nodeLabel.classed('hidden'),
+      showLabels: nodeLabel._groups[0].length > 0 ? !nodeLabel.classed('hidden') : null,
```

Comment:
What is `_groups`? Does it always contain at least one element (`[0]`)?

Reply:
``_groups`` is a property of d3 object, which contains (in our case) only one list containing all labels to be generated in the graph. If the list is empty, we should not execute ``.classed()`` function on the object, which occures with the error. In my code I wanted to stick to this object and check if it contains any values to display.

We could also think about checking if ``nodes`` list is empty, but ... we don't have guarantee that both values are empty in the same time in any case (for example some library bugs in corner cases).